host/src/FunctionsNetHost/Logger.cs

```diff
@@ -17,43 +16,30 @@ static Logger()
 #else
             LogPrefix = "";
 #endif
-            string traceLoggingEnabled =
-                Environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionsNetHostTrace) ?? "0";
-            IsDebugLevelLogEnabled = traceLoggingEnabled == "1";
         }
 
-        internal static bool IsDebugLogEnabled => IsDebugLevelLogEnabled;
-
-        /// <summary>
-        /// Logs a debug message if "AZURE_FUNCTIONS_FUNCTIONSNETHOST_TRACE" environment variable value is set to "1"
-        /// For optimal performance (when building the log message needs execute some code (Ex: someObj.ToString() or so),
-        /// consider checking "IsDebugLogEnabled" before calling LogDebug.
-        /// </summary>
-        internal static void LogDebug(string message)
+        internal static bool IsTraceLogEnabled
```

Comment:
Simplified this Logger to only 2 methods and one property to expose whether trace level logging is enabled.

In the previous version I cached the environment variable value (to determine whether trace logging should be enabled or not). That would not work if customer wants to enable this setting via portal (configuration->settings). If we observe an issue with calls to environment variable read, we can revisit.